ui/MultiSigDialogs.py

```diff
@@ -977,11 +975,11 @@ def createLockboxDashboardTab(self):
       # simulfunding widgets
       self.stkDashboard = QStackedWidget()
 
-      simultxt = 'Simul'
+      simultxt = self.tr('SimulFund')
```

Comment:
It came up in a transifex issue where the translator was saying that some of the strings say "check the Simul box" (or something like that) but Simul was not in the strings to be translated.